src/parachain/redeem.ts

```diff
@@ -118,16 +123,33 @@ export interface RedeemAPI {
      * and required completion time by a user.
      */
     getRedeemPeriod(): Promise<BlockNumber>;
+    /**
+     * Send a transaction to set the DOT/BTC exchange rate
+     * @param amount The amount of PolkaBTC to burn, denominated as PolkaBTC
+     */
+    burn(amount: Big): Promise<void>;
+    /**
+     * @returns The maximum amount of tokens that can be burned through a liquidation redeem
+     */
+    getMaxBurnableTokens(): Promise<Big>;
+    /**
+     * @returns The exchange rate (collateral currency to wrapped token currency)
+     * used when burning tokens. This exchange rate is at least as favourable to the 
```

Comment:
Actually that is not always the case. In massive dot exchange rate crashes we might end up with lower dot amounts than you get for burning polkabtc